src/components/browse/singleDatasetView/singleDatasetViewTabs/singleDatasetViewTabs.tsx

```diff
@@ -24,23 +24,27 @@ const SingleDatasetViewTabs = (props: SingleDatasetViewTabsProps) => {
           <Nav.Item>
             <Nav.Link
               eventKey="0"
-              className="border border-1 mx-2 border-light"
+              className="border border-1 mx-2 border-light text-center"
```

Comment:
Could you change the box border color as in the design? It should be EFF0F6, and the dropdown shadow

Reply:
We need to define a fixed palette. If this is what you want the shadow and `Light` colour to be, then I can make it so.

Reply:
could be nicer